internal/monitors/traefik/metadata.yaml

```diff
@@ -0,0 +1,297 @@
+    typically passed in as a command line argument.
+
+    `./traefik -c traefik.toml`
+
+    However, when running the Traefik Docker image, the configuration file is mounted to the
+    volume `/etc/traefik/traefik.toml`. For example,
+
+    `docker run -d -p 8080:8080 -p 80:80 -v $PWD/traefik.toml:/etc/traefik/traefik.toml`
+
+    If the configuration file is not available use the
+    <a target="_blank" href="https://raw.githubusercontent.com/containous/traefik/master/traefik.sample.toml">sample
+    configuration file</a>.
+    The Smart Agent must have network access to Traefik. This means for instance that, in a Docker environment,
+    the Smart Agent container needs to share a network with the Traefik container. So the Docker run command
+    to start the agent may look as follows:
+
+    Before we can send metrics to SignalFx, we need to make sure you are sending them to the correct SignalFx realm.
+    To determine what realm you are in (YOUR_SIGNALFX_REALM), check your profile page in the SignalFx web application
+    (click the avatar in the upper right and click My Profile).
+    If you are not in the `us0` realm, you will need to set the `URL` configuration option below to use the
+    correct realm, as shown below.
+
+    You will also need to set the `SFX_ACCESS_TOKEN` configuration option to your SignalFx organization access token
+    (YOUR_SIGNALFX_API_TOKEN).
+    For more information on authentication, see the API's
+    [Authentication documentation](https://developers.signalfx.com/basics/authentication.html).
+    ```
+    docker run --rm \
+    -e SFX_ACCESS_TOKEN=YOUR_SIGNALFX_API_TOKEN \
+    -e SFX_INGEST_URL=https://ingest.YOUR_SIGNALFX_REALM.signalfx.com \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v <agent.yaml path>:/etc/signalfx/agent.yaml \
+    --net <traefik container network> \
+    quay.io/signalfx/signalfx-agent:<version>
+    ```
+    #### Smart Agent Configuration
+
+    Find and edit the Smart Agent configuration file `agent.yaml` to enable the Docker observer and configure the
+    prometheus-exporter monitor as described
+    <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/9feb3f77fdf6de46dc476f62568ad4f9b725660c/docs/monitors/prometheus-exporter.md">here</a>.
+    For example, the configuration below will cause the Smart Agent to query the Docker Engine API for all running
+    containers with port 8080 exposed and export Prometheus metrics from endpoint `<container ip>:8080/metrics`.
+
+    ```
+    observers:
+    - type: docker
+    monitors:
+    - type: prometheus-exporter
+      discoveryRule: port == 8080
+    extraDimensions:
+      metric_source: traefik
+    ```
+    Also, metrics can be excluded by configuring the `metricsToExclude` section of the Smart Agent configuration file.
+
+    ### USAGE
+
+    The Smart Agent exports Prometheus metrics exposed by Traefik. These metrics can be categorized into Traefik-related,
```

Comment:
This should read "Traefik exports Prometheus metrics that can be scraped by the Smart Agent.".

Also, this section should be the first section above under "Description" (get rid of the "Description" header though as well).